FCI_FYP_Template/Makefile

```diff
@@ -0,0 +1,9 @@
+all:
+	pdflatex thesis
+	bibtex thesis
+	pdflatex thesis
+	pdflatex thesis
+
+ifeq ($(shell uname -s), Darwin)
```

Comment:
If user is on mac, it will open the PDF right away with Preview.app. Maybe we can find similar things for Linux users.